test/CarouselSpec.js

```diff
@@ -37,7 +37,7 @@ describe('<Carousel />', () => {
     const fixedItem = <span>Do you rock!?</span>;
     wrapper = renderer({ images, fixedItem });
     expect(wrapper.find('.carousel-fixed-item').length).to.eq(1);
-    expect(wrapper.find('.carousel-fixed-item').childAt(0).node).to.eq(fixedItem);
+    expect(wrapper.find('.carousel-fixed-item').childAt(0).equals(fixedItem)).to.eq(true);
```

Comment:
Why is this change?

Reply:
This is a breaking change from `enzyme`: https://github.com/airbnb/enzyme/blob/master/docs/guides/migration-from-2-to-3.md#private-properties-and-methods-have-been-removed

Reply:
Good catch, I'll have to keep an eye out for that elsewhere.